src/appConfigs/schema.ts

```diff
@@ -14,6 +17,20 @@ export const typeDefs = gql`
     toUid: ID!
   }
 
+  type ContactWith {
+    uid: ID!
+    met_on: String!
+    risk: String!
+  }
+
+  type ContactWithList {
+    metfriendon: String!
```

Comment:
It took me a while to figure out the difference between `metfriendon` and `date`. How could we name this to make it clearer?

Reply:
Ignore this for now. not used this yet. will use later.